x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -563,65 +566,71 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
                     bw.newLine();
                     bw.write(AUTO_CONFIGURATION_START_MARKER);
                     bw.newLine();
-                    bw.write("###################################################################################");
+                    bw.write("#");
                     bw.newLine();
-                    bw.write("# The following settings, and associated TLS certificates and keys configuration, #");
+                    bw.write("# The following settings, TLS certificates, and keys have been automatically      ");
                     bw.newLine();
-                    bw.write("# have been automatically generated in order to configure Security.               #");
+                    bw.write("# generated to configure Elasticsearch security features on ");
+                    bw.write(autoConfigDate.format(dateTimeFormatter));
                     bw.newLine();
-                    bw.write("# These have been generated the first time that the new node was started, only    #");
+                    bw.write("#");
+                    // TODO add link to docs
                     bw.newLine();
-                    bw.write("# if Security had not been explicitly configured beforehand.                      #");
+                    bw.write("# --------------------------------------------------------------------------------");
                     bw.newLine();
-                    bw.write(String.format(Locale.ROOT, "# %-79s #", ""));
                     bw.newLine();
-                    bw.write(String.format(Locale.ROOT, "# %-79s #", autoConfigDate));
-                    // TODO add link to docs
+                    bw.write("# Security features are enabled");
```

Comment:
```suggestion
                    bw.write("# Enable security features");
```

What about this? It sounds more consistent with all the other comments.

Reply:
sure thing